staging/src/k8s.io/code-generator/cmd/informer-gen/generators/packages.go

```diff
@@ -281,7 +281,7 @@ func groupPackage(basePackage string, groupVersions clientgentypes.GroupVersions
 	groupPkgName := strings.Split(string(groupVersions.PackageName), ".")[0]
 
 	return &generator.DefaultPackage{
-		PackageName: groupPkgName,
+		PackageName: genutil.NormalizeGoPackageName(groupPkgName),
```

Comment:
does this only apply to informer-gen? Why are the others not affected?

Reply:
Hye, according to git bisect, no. I will double for client-test.

Reply:
hye, is it possible to approve the pull request please ?